dts/bindings/haptics/ti,drv2605.yaml

```diff
@@ -26,10 +27,29 @@ properties:
       - "8X"
       - "16X"
       - "DISABLED"
+    default: "4X"
```

Comment:
Please document default values 
https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#rules-for-default-values